_posts/2018-09-29-cfp-in-progress.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Call for Proposals in progress, keynotes announcement"
+authors:
+  - skade
+  - apiraino
```

Comment:
who are the authors? (funny question, I know)

Reply:
not me, you